settings/src/components/AuthSecurityPassword.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+    <div id="security-password" class="section">
+        <h2 class="inlineblock">{{ t('settings','Password') }}</h2>
+        <span id="password-error-msg" class="msg success hidden">Saved</span>
+        <div class="personal-settings-setting-box personal-settings-password-box">
+			<form id="passwordform" @submit.prevent>
+                <label for="pass1" class="hidden-visually">{{ t('settings','Current password') }} </label>
+				<input type="password" id="pass1" name="oldpassword"
+                    :placeholder="t('settings','Current password')"
+                    :value="oldpass"
+                    autocomplete="off" autocapitalize="none" autocorrect="off" />
+
+                <v-popover trigger="manual" :open="IsOpen" offset="3" :auto-hide="false">
+                    <div class="personal-show-container">
+                        <label for="pass2" class="hidden-visually">{{ t('settings','New password') }}</label>
+                        <input type="password" id="pass2" name="newpassword"
+                            :placeholder="t('settings','New password')"
+                            :value="newpass"
+                            data-typetoggle="#personal-show"
+                            autocomplete="off" autocapitalize="none" autocorrect="off" />
+                        
+                        <password :value="newpass" strengthMeterClass = "strengthMeterClass" 
```

Comment:
no spaces around `=`
And no PascalCase in attributes (vue will automatically convert the data to match the prop) ;)

```suggestion
                        <password :value="newpass" strength-meter-class="strengthMeterClass" 
```